migrations/versions/0328_purge_broadcast_data.py

```diff
@@ -0,0 +1,24 @@
+"""
+
+Revision ID: 0328_purge_broadcast_data
+Revises: 0327_idx_notification_history
+Create Date: 2020-09-02 17:16:27.545673
+
+"""
+from alembic import op
+
+
+revision = '0328_purge_broadcast_data'
+down_revision = '0327_idx_notification_history'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("TRUNCATE broadcast_event, broadcast_message;")
```

Comment:
This will get run before the rest of the code is deployed, which is fine but very small risk that we truncate the table, code begins deploying out, a user creates a broadcast with the old schema, code finishes getting deployed out. This would leave us with the old schema of the data in the DB which would need a manual tidy up. Very happy to tolerate that risk however, just thought I'd point it out just in case you weren't aware of it.

Reply:
Thanks, I do realise there are a few things in those deploys that would be risky if the feature was live. As it is, the feature is not live yet and I plan to ask our team to not use the broadcast feature while I deploy.